src/Model/SerializerWithBuiltIns.coffee

```diff
@@ -0,0 +1,29 @@
+_ = require "underscore"
+Dataflow = require "../Dataflow/Dataflow"
+Model = require "./Model"
+Util = require "../Util/Util"
+Storage = require "../Storage/Storage"
+
+
+module.exports = SerializerWithBuiltIns = {}
```

Comment:
This code (gathering built-in objects and building a serializer using them) used to be in `Editor.coffee`. But the viewer also needs the same serializer, to deserialize diagrams made by the editor.